lib/navigation.dart

```diff
@@ -0,0 +1,76 @@
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:streamer/home.dart';
+import 'package:streamer/pages/Songs_List.dart';
+import 'package:streamer/subsonic/subsonic.dart';
+
+class Navigation extends StatefulWidget {
+  final SubsonicContext subSonicContext;
+
+  const Navigation({super.key, required this.subSonicContext});
+
+  @override
+  State<Navigation> createState() => _Navigation();
+}
+
+class _Navigation extends State<Navigation> {
+  int _selectedTab = 0;
+
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  Widget _children(SubsonicContext context, int index) {
+    List<Widget> children = [
+      SongsList(subSonicContext: context),
+      Home(subSonicContext: context),
+      // Augusto please add your settings page here
```

Comment:
@aumfreire If you want you could try merging this branch into your settings, and try to have settings and navigation bar linked to each other.